spec/features/providers/user_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+feature "Provider users page" do
+  let(:provider_users_page) { PageObjects::Page::Providers::Users::IndexPage.new }
+  let(:new_access_request_page) { PageObjects::Page::Organisations::NewManualAccessRequestPage.new }
+
+  let(:current_recruitment_cycle) { build(:recruitment_cycle) }
+
+  scenario "View a provider's user page" do
```

Comment:
Just being pedantic, but this is testing more than just viewing the user's page, it's testing the request access journey. :)

Reply:
True ... We are testing that clicking on the link takes us to the 'request access' page but we do not go any further after that because request access functionality is tested elsewhere. As this is a first iteration there is still lot of functionality missing from the page. Would be worth revisiting the specs once it's fleshed out a bit more.